src/ui/public/modals/confirm_modal.html

```diff
@@ -1,4 +1,16 @@
 <div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
+  <div class="kuiModalHeader" ng-if="title">
```

Comment:
Yeah, you're right. I changed my mind. Still, there's a weird conflict if someone tries to set `showClose` to true, but doesn't specify a title. Since the component doesn't support that, we should let the engineer know they're misusing the interface. How about throwing an error?